MojangAPI.fetchSkinProfile(uuid)!!.profileProperties() 에러!!

[17:56:22 ERROR]: Could not pass event PlayerInteractEvent to ServerManagement v1.0-SNAPSHOT
java.util.concurrent.ExecutionException: kotlinx.serialization.json.internal.JsonDecodingException: Unexpected JSON token at offset 1262: Encountered an unknown key 'profileActions' at path: $.properties
Use 'ignoreUnknownKeys = true' in 'Json {}' builder to ignore unknown keys.
JSON input: .....30+71awLoJj7xozFw="
  } ],
  "profileActions" : { }
}
        at java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396) ~[?:?]
        at java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2073) ~[?:?]
        at com.github.cozy06.Main.onPlayerInteract(Main.kt:91) ~[ServerManagement-1.0-SNAPSHOT.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor2.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor$2.execute(EventExecutor.java:77) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:81) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:git-Paper-96]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperEventManager.callEvent(PaperEventManager.java:54) ~[paper-1.20.1.jar:git-Paper-96]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.callEvent(PaperPluginManagerImpl.java:126) ~[paper-1.20.1.jar:git-Paper-96]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:615) ~[paper-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_20_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:566) ~[paper-1.20.1.jar:git-Paper-96]
        at org.bukkit.craftbukkit.v1_20_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:523) ~[paper-1.20.1.jar:git-Paper-96]
        at org.bukkit.craftbukkit.v1_20_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:517) ~[paper-1.20.1.jar:git-Paper-96]
        at org.bukkit.craftbukkit.v1_20_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:513) ~[paper-1.20.1.jar:git-Paper-96]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.handleAnimate(ServerGamePacketListenerImpl.java:2619) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundSwingPacket.handle(ServerboundSwingPacket.java:25) ~[?:?]
        at net.minecraft.network.protocol.game.ServerboundSwingPacket.a(ServerboundSwingPacket.java:11) ~[?:?]
        at net.minecraft.network.protocol.PacketUtils.lambda$ensureRunningOnSameThread$0(PacketUtils.java:51) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.20.1.jar:git-Paper-96]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1339) ~[paper-1.20.1.jar:git-Paper-96]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[paper-1.20.1.jar:git-Paper-96]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1316) ~[paper-1.20.1.jar:git-Paper-96]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1309) ~[paper-1.20.1.jar:git-Paper-96]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1287) ~[paper-1.20.1.jar:git-Paper-96]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1175) ~[paper-1.20.1.jar:git-Paper-96]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:318) ~[paper-1.20.1.jar:git-Paper-96]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: kotlinx.serialization.json.internal.JsonDecodingException: Unexpected JSON token at offset 1262: Encountered an unknown key 'profileActions' at path: $.properties
Use 'ignoreUnknownKeys = true' in 'Json {}' builder to ignore unknown keys.
JSON input: .....30+71awLoJj7xozFw="
  } ],
  "profileActions" : { }
}
        at kotlinx.serialization.json.internal.JsonExceptionsKt.JsonDecodingException(JsonExceptions.kt:24) ~[?:?]
        at kotlinx.serialization.json.internal.JsonExceptionsKt.JsonDecodingException(JsonExceptions.kt:32) ~[?:?]
        at kotlinx.serialization.json.internal.AbstractJsonLexer.fail(AbstractJsonLexer.kt:530) ~[?:?]
        at kotlinx.serialization.json.internal.AbstractJsonLexer.failOnUnknownKey(AbstractJsonLexer.kt:525) ~[?:?]
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.handleUnknown(StreamingJsonDecoder.kt:251) ~[?:?]
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeObjectIndex(StreamingJsonDecoder.kt:237) ~[?:?]
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeElementIndex(StreamingJsonDecoder.kt:172) ~[?:?]
        at io.github.monun.tap.mojangapi.MojangAPI$SkinProfile$$serializer.deserialize(MojangAPI.kt:64) ~[?:?]
        at io.github.monun.tap.mojangapi.MojangAPI$SkinProfile$$serializer.deserialize(MojangAPI.kt:64) ~[?:?]
        at kotlinx.serialization.json.internal.StreamingJsonDecoder.decodeSerializableValue(StreamingJsonDecoder.kt:70) ~[?:?]
        at kotlinx.serialization.json.Json.decodeFromString(Json.kt:95) ~[?:?]
        at io.github.monun.tap.mojangapi.MojangAPI$fetchSkinProfileAsync$$inlined$fetchAsync$1.invoke(MojangAPI.kt:121) ~[?:?]
        at io.github.monun.tap.mojangapi.MojangAPI$fetchSkinProfileAsync$$inlined$fetchAsync$1.invoke(MojangAPI.kt:53) ~[?:?]
        at io.github.monun.tap.mojangapi.MojangAPI$sam$java_util_function_Function$0.apply(MojangAPI.kt) ~[?:?]
        at java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:646) ~[?:?]
        at java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510) ~[?:?]
        at java.util.concurrent.CompletableFuture.postFire(CompletableFuture.java:614) ~[?:?]
        at java.util.concurrent.CompletableFuture$UniWhenComplete.tryFire(CompletableFuture.java:844) ~[?:?]
        at java.util.concurrent.CompletableFuture$Completion.exec(CompletableFuture.java:483) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
        at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
        at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]

라는 에러가 뜨네요.....ㅠㅠ

--- a/tap-api/src/main/kotlin/io/github/monun/tap/mojangapi/MojangAPI.kt
+++ b/tap-api/src/main/kotlin/io/github/monun/tap/mojangapi/MojangAPI.kt
@@ -52,7 +52,7 @@ object MojangAPI {
 
         return client.sendAsync(request, HttpResponse.BodyHandlers.ofString()).thenApply { response ->
             val body = response.body()
-            if (body.isBlank()) null else Json.decodeFromString<T>(body)
+            if (body.isBlank()) null else Json { ignoreUnknownKeys = true }.decodeFromString<T>(body)
         }
     }
 
@@ -69,7 +69,7 @@ object MojangAPI {
     ) {
         fun textureProfile() = properties.find { it.name == "textures" }?.let { textures ->
             val string = Base64.getDecoder().decode(textures.value).decodeToString()
-            Json.decodeFromString<TextureProfile>(string)
+            Json { ignoreUnknownKeys = true }.decodeFromString<TextureProfile>(string)
         }
 
         fun profileProperties() = properties.map { it.profileProperty() }

